cli/src/Morphir/Web/DevelopApp.elm

```diff
@@ -1815,6 +1983,28 @@ viewDefinitionDetails model =
                                                                             ]
                                                                   }
                                                                 , { name = "XRay View", content = XRayView.viewValueDefinition (XRayView.viewType <| pathToUrl) valueDef }
+                                                                , { name = "Custom Attributes"
+                                                                  , content =
+                                                                        row
+                                                                            [ width fill
+                                                                            , height fill
+                                                                            , spacing
+                                                                                (model.theme
+                                                                                    |> Theme.scaled 8
+                                                                                )
+                                                                            , paddingEach
+                                                                                { left =
+                                                                                    model.theme
+                                                                                        |> Theme.scaled 2
+                                                                                , top = 0
+                                                                                , right =
+                                                                                    model.theme
+                                                                                        |> Theme.scaled 2
+                                                                                , bottom = 0
+                                                                                }
+                                                                            ]
```

Comment:
O don't think we should have individual paddings defined for each tab. It feels like that should be done in the tab component itself. @bekand, what do you think?

Reply:
I had a chat with @bekand about this and this looks generally ok. The only think I would change is to use `paddingXY` instead of `paddingEach`.